mne/report.py

```diff
@@ -965,6 +967,96 @@ def add_bem_to_section(self, subject, caption='BEM', section='bem',
         self._sectionlabels.append(sectionvar)
         self.html.extend(html)
 
+    def add_slider_to_section(self, figs, captions=None, section='custom',
+                              scale=None, image_format='png'):
+        """Renders a slider of figs to the report.
+
+        figs : list of figures.
+            Each figure in the list can be an instance of
+            matplotlib.pyplot.Figure, mayavi.core.scene.Scene,
+            or np.ndarray (images read in using scipy.imread).
+        captions : list of str | float | None
+            A list of captions to the figures. If float, a str will be
+            constructed as `Data: %f ms`. If None, it will default to
+            `Data slice %d`.
+        section : str
+            Name of the section. If section already exists, the figures
+            will be appended to the end of the section
+        scale : float | None | callable
+            Scale the images maintaining the aspect ratio.
+            If None, no scaling is applied. If float, scale will determine
+            the relative scaling (might not work for scale <= 1 depending on
+            font sizes). If function, should take a figure object as input
+            parameter. Defaults to None.
+        image_format : {'png', 'svg'}
+            The image format to be used for the report. Defaults to 'png'.
```

Comment:
the svg output was behaves weirdly on chrome + ubuntu: there's an extra image frame.

On firefox + Ubuntu, it works fine but there's an extra `"> `